addons/website/static/tests/tours/snippet_popup_and_scrollbar.js

```diff
@@ -37,7 +37,7 @@ wTourUtils.registerWebsitePreviewTour("snippet_popup_and_scrollbar", {
     test: true,
     url: "/",
     edition: true,
-}, () => [
+}, () => wTourUtils.noBusyModal([
```

Comment:
I did initially write it by adding the intermediate step between each step that caused the problem - I introduced this notation to avoid having to do that. Pretty much every step that showed or hid a popup caused the problem - but it only made much further steps fail.